src/renderers/testing/ReactTestMount.js

```diff
@@ -20,8 +20,8 @@ var getHostComponentFromComposite = require('getHostComponentFromComposite');
 var instantiateReactComponent = require('instantiateReactComponent');
 var invariant = require('invariant');
 
-type TestRendererOptions = {
-  createNodeMock: (element: ReactElement) => Object,
+export type TestRendererOptions = {
+  createNodeMock: (element: ReactElement<any>) => Object,
```

Comment:
Please change `Object` to either `any`, `mixed` or the proper type.

The problem with using `Object` is that it makes it look like it is flow typed but in practice flow doesn't know any of its attributes so it cannot verify anything about it.

We're trying to either have the correct type or `mixed` (and if `mixed` doesn't work then `any`)